packages/react-devtools-shared/src/__tests__/treeContextStateSerializer.js

```diff
@@ -2,15 +2,14 @@ import {printStore} from 'react-devtools-shared/src/devtools/utils';
 
 // test() is part of Jest's serializer API
 export function test(maybeState) {
-  if (maybeState === null || typeof maybeState !== 'object') {
-    return false;
-  }
-
```

Comment:
This change seems arbitrary. You added this to `packages/react-devtools-shared/src/__tests__/inspectedElementSerializer.js` and removed it here. Why?

Reply:
I did? At some point during rebasing I thought about normalizing both serializers but apparently I didn't. The goal was to have both look the same so I'll make sure that actually happens.

Reply:
Yeah. I think you added a null/object check to the start of the test method there and removed it here.

Reply:
Restored the early return. `treeContextStateSerializer` and `dehydratedValueSerializer` should now follow the same pattern.